specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlPool.json

```diff
@@ -4459,6 +4975,149 @@
           "type": "string"
         }
       }
+    },
+    "WorkloadGroupProperties": {
+      "description": "Workload group definition. For more information look at sys.workload_management_workload_groups (DMV).",
+      "required": [
+        "minResourcePercent",
+        "maxResourcePercent",
+        "minResourcePercentPerRequest"
+      ],
+      "type": "object",
+      "properties": {
+        "minResourcePercent": {
+          "format": "int32",
+          "description": "The workload group minimum percentage resource.",
+          "type": "integer"
+        },
+        "maxResourcePercent": {
+          "format": "int32",
+          "description": "The workload group cap percentage resource.",
+          "type": "integer"
+        },
+        "minResourcePercentPerRequest": {
+          "format": "double",
+          "description": "The workload group request minimum grant percentage.",
+          "type": "number"
+        },
+        "maxResourcePercentPerRequest": {
+          "format": "double",
+          "description": "The workload group request maximum grant percentage.",
+          "type": "number"
+        },
+        "importance": {
```

Comment:
can this be a string enum?  How does the user know what values are accepted here?

Reply:
Hi, The Synapse RP API for which this documentation is being generated is like a proxy level that calls into SQL API that already exists today and is in use. 
So changing the SQL API  to change this property to an enum is a bigger change. 
Is it ok if we describe the importance levels like this:
There are five levels of importance: low, below_normal, normal, above_normal, and high. (from here : https://docs.microsoft.com/en-us/azure/synapse-analytics/sql-data-warehouse/sql-data-warehouse-workload-importance )